### PR TITLE
Add vulkan-validation-layers

### PR DESCRIPTION
**Pull request by JarrettSJohnson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JarrettSJohnson** [comment]
Ready for review, @conda-forge/help-c-cpp.

File: recipes/vulkan-validation-layers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  host:
+    - libxcb             # [linux]
+    - xorg-libx11        # [linux]
+    - xorg-libxrandr     # [linux]
+    - wayland            # [linux]
+    - wayland-protocols  # [linux]
+
+test:
+  commands:
+    - if not exist %LIBRARY_PREFIX%\\bin\\VkLayer_khronos_validation.dll exit 1  # [win]
```
Comment (xhochy) [inline recipes/vulkan-validation-layers/meta.yaml:31]: Please also check for the library on Linux

File: recipes/vulkan-validation-layers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 37fd611f1c5c5879a68c6f63b09e600a3858f519aa147a2535961fa3d8e889ae
+
+build:
```
Comment (xhochy) [inline recipes/vulkan-validation-layers/meta.yaml:12]: Please add a `run_exports`

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by JarrettSJohnson** [comment]
Ready for re review,  @conda-forge/help-c-cpp .

File: recipes/vulkan-validation-layers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - wayland            # [linux]
+    - wayland-protocols  # [linux]
+
+test:
+  commands:
+    - if not exist %LIBRARY_PREFIX%\\bin\\VkLayer_khronos_validation.dll exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\\bin\\VkLayer_khronos_validation.json exit 1  # [win]
+    - test -d $PREFIX/share/vulkan/explicit_layer.d/  # [not win]
+    - test -f $PREFIX/share/vulkan/explicit_layer.d/VkLayer_khronos_validation.json  # [not win]
+    - test -f $PREFIX/lib/libVkLayer_khronos_validation.so  # [linux]
```
Comment (xhochy) [inline recipes/vulkan-validation-layers/meta.yaml:37]:
```suggestion
    - test -f $PREFIX/lib/libVkLayer_khronos_validation${SHLIB_EXT}  # [unix]
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**